gui/src/components/dashboard/capacity-gauge.vue

```diff
@@ -68,10 +68,10 @@ export default class CortxCapacityGauge extends Vue {
       .dispatch("performanceStats/getCapacityStats")
       .then(capacityC3Data => {
         this.chartDataVal = capacityC3Data[0][1] ? capacityC3Data[0][1] : 0.00;
-        if (this.chartDataVal <= 50.00) {
+        if (this.chartDataVal < 50.00) {
```

Comment:
As a fallback we should convert `chartDataVal` to floating value.

e.g ->`parseFloat(this.chartDataVal).toFixed(2)`

Reply:
why do we want to parse float? even why do we want to 90.00 or 50.00
we should simply check for 50
![image](https://user-images.githubusercontent.com/66459168/96856158-3b199f80-147b-11eb-94d4-710edb9d0f46.png)

Reply:
resolved the review comment checked for only number